.circleci/config.yml

```diff
@@ -0,0 +1,117 @@
+aliases:
+  - &restore_sbt_cache
+    key: sbt-cache-{{ checksum "/tmp/scala_version" }}
+
+  - &save_sbt_cache
+    key: sbt-cache-{{ checksum "/tmp/scala_version" }}-{{ epoch }}
+    paths:
+      - "~/.ivy2/cache"
+      - "~/.sbt"
+      - "~/.cache/coursier"
+
+  - &run_cibuild
+    - checkout
+    - run: echo "${SCALA_VERSION}" > /tmp/scala_version
+    - restore_cache: *restore_sbt_cache
+    - run:
+        name: Executing cibuild
+        command: ./scripts/cibuild
+    - save_cache: *save_sbt_cache
+
+  - &run_cipublish
+    - checkout
+    - run: echo "${SCALA_VERSION}" > /tmp/scala_version
+    - restore_cache: *restore_sbt_cache
+    - run:
+        name: "Import signing key"
+        command: |
+          echo "${GPG_KEY}" | base64 -d > signing_key.asc && \
+          gpg --batch \
+            --passphrase "${GPG_PASSPHRASE}" \
+            --import signing_key.asc
+    - run:
+        name: Executing cipublish
+        command: ./scripts/cipublish
+
+  # Build environments
+  - &machine-openjdk8-scala2_11_12-environment
+    machine:
+      image: ubuntu-1604:201903-01
+    environment:
+      SCALA_VERSION: 2.11.12
+
+  - &machine-openjdk8-scala2_12_8-environment
+    machine:
+      image: ubuntu-1604:201903-01
+    environment:
+      SCALA_VERSION: 2.12.8
+  - &openjdk8-scala2_11_12-environment
+    docker:
+      - image: circleci/openjdk:8-jdk
+    environment:
+      SCALA_VERSION: 2.11.12
+
+  - &openjdk8-scala2_12_8-environment
+    docker:
+      - image: circleci/openjdk:8-jdk
+    environment:
+      SCALA_VERSION: 2.12.8
+
+version: 2
+workflows:
+  version: 2
+  build:
+    jobs:
+      - "openjdk8-scala2.11.12":
+          filters: # required since `openjdk8-scala2.11.12_deploy` has tag filters AND requires `openjdk8-scala2.11.12`
+            tags:
+              only:
+                - /^(.*)$/
+      - "openjdk8-scala2.12.8":
+          filters: # required since `openjdk8-scala2.12.8_deploy` has tag filters AND requires `openjdk8-scala2.12.8`
+            tags:
+              only:
+                - /^(.*)$/
+      - "openjdk8-scala2.11.12_deploy":
+          requires:
+            - "openjdk8-scala2.11.12"
+          filters:
+            tags:
+              only:
+                - /^(.*)$/
+            branches:
```

Comment:
If you want to test publishing without merging this PR, just add this branch name to this array and in the same array for the 2.12 deploy below.

I'm also noticing that this branch is on a fork. You'll likely need to push your branch to geotrellis/vectorpipe after making the noted change.